client/src/components/EditProfileForm.js

```diff
@@ -172,30 +177,48 @@ const EditProfileForm = ({ onSubmit }) => {
         <Typography component="h6" variant="h6">
           Favorite Cuisines:
         </Typography>
-        {cuisines.map((cuisine) => {
-          const isSelected = selectedCuisines.includes(cuisine.id);
-          return (
-            <Button
-              className={classes.button}
-              color={isSelected ? "primary" : "default"}
-              variant="contained"
-              key={cuisine.id}
-              onClick={() =>
-                isSelected
-                  ? setSelectedCuisines(
-                      selectedCuisines.filter(
-                        (cuisineId) => cuisineId !== cuisine.id
+        <Box>
+          {cuisines.map((cuisine) => {
```

Comment:
Add a conditional here so favorite cuisines do not show if user is a chef.

Reply:
how about when I am a chef and i want to change my favorite cuisines as a user?